docs/en/connector-v2/source/StarRocks.md

```diff
@@ -0,0 +1,144 @@
+# StarRocks
+
+> StarRocks source connector
+
+## Description
+Read external data source data through StarRocks.
+The internal implementation of StarRocks source connector is obtains the query plan from the frontend (FE), 
+delivers the query plan as a parameter to BE nodes, and then obtains data results from BE nodes..
+## Key features
+
+- [x] [batch](../../concept/connector-v2-features.md)
+- [ ] [stream](../../concept/connector-v2-features.md)
+- [x] [exactly-once](../../concept/connector-v2-features.md)
+- [x] [schema projection](../../concept/connector-v2-features.md)
+
+## Options
+
+| name                    | type   | required | default value     |
+|-------------------------|--------|----------|-------------------|
+| node_urls               | list   | yes      | -                 |
+| username                | string | yes      | -                 |
+| password                | string | yes      | -                 |
+| database                | string | yes      | -                 |
+| table                   | string | yes      | -                 |
+| scan_filter             | string | no       | -                 |
+| fields                  | config | yes      | -                 |
```

Comment:
For the newly added Connector, we use the scheam field to define this information. FYI https://github.com/apache/incubator-seatunnel/issues/3823

Reply:
here

Reply:
> For the newly added Connector, we use the scheam field to define this information. FYI #3823
thanks，already fix it.PTAL